common/static/js/vendor/pdfjs/viewer.html

```diff
@@ -108,9 +108,9 @@
               <span data-l10n-id="download_label">Download</span>
             </button>
 
-            <a href="#" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" tabindex="55" data-l10n-id="bookmark">
+            <button type="button" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" tabindex="55" data-l10n-id="bookmark">
```

Comment:
Can you back out this change and create a ticket specifically for this issue? We need to come to agreement about how to handle vendor file changes.